libraries/Update/src/HttpsOTAUpdate.h

```diff
@@ -0,0 +1,12 @@
+#include "esp_http_client.h"
+#define HttpEvent_t esp_http_client_event_t
+class HttpsOTAUpdateClass {
+
+    public:
+    void start_firmware_upgrade(const char * cert, const char * url, bool skip_cert_cn_check);
+    void onHttpEvent(void (*ptr)(HttpEvent_t *));
+    bool ota_successful();
+    bool ota_fail();
```

Comment:
Some function names in Camel case and some in Snake case? Can you please maintain consistency as per the ESP32 Arduino standards?